test/unit/support/shared/base_context.rb

```diff
@@ -12,6 +12,9 @@
     # Create a thing to store our temporary files so that they aren't
     # unlinked right away.
     @_temp_files = []
+
+    # Roughly simulate the embedded Bundler availability
+    $vagrant_bundler_runtime = Object.new
```

Comment:
Please let me know if this is too ugly, and whether this should be implemented as a Mock/Stub object